NYPLCardCreator/Classes/AddressViewController.swift

```diff
@@ -114,6 +114,16 @@ final class AddressViewController: FormTableViewController {
     self.zipCell.textField.addTarget(self,
                                      action: #selector(zipTextFieldDidChange),
                                      for: .allEditingEvents)
+
+    // grouping these together to do 1 OS check for all these fields at once
+    // for auto-fill feature
```

Comment:
You've done a fine job self-documenting this code, so the extra comment isn't necessary.